crates/sui-keys/src/keystore.rs

```diff
@@ -80,7 +101,7 @@ pub trait AccountKeystore: Send + Sync {
         old_alias: &str,
         new_alias: Option<&str>,
     ) -> Result<String, anyhow::Error> {
-        if !self.alias_exists(old_alias) {
+        if !self.aliases().iter().any(|a| a.alias == old_alias) {
```

Comment:
I'm not a big fan of this decision -- it means that regardless of whether the implementor has an efficient way to check whether an alias exists, the trait forces us into using a linear scan.